mllib/src/main/scala/org/apache/spark/ml/feature/HashingTF.scala

```diff
@@ -137,22 +146,27 @@ class HashingTF @Since("1.4.0") (@Since("1.4.0") override val uid: String)
 @Since("1.6.0")
 object HashingTF extends DefaultParamsReadable[HashingTF] {
 
+  private[ml] val HASH_FUNC_VERSION_1 = 1
+  private[ml] val HASH_FUNC_VERSION_2 = 2
```

Comment:
Seems redundant to declare these if they're just ints.
I was thinking strings might be more descriptive, but this is OK.